include/capstone/ppc.h

```diff
@@ -148,9 +148,9 @@ static inline ppc_br_hint PPC_get_hint(uint8_t bo) {
 	if (!DecrCTR && !TestCR)
 		return PPC_BR_NOT_GIVEN;
 	else if (DecrCTR && !TestCR)
-		return ((bo & PPC_BO_CR_CMP) >> 2) | (bo & PPC_BO_T);
+               return (ppc_br_hint)(((bo & PPC_BO_CR_CMP) >> 2) | (bo & PPC_BO_T));
```

Comment:
Please use tabs here. Also below.

Reply:
Sorry, patch got mangled when I transferred it between machines.  Fixed.